.azure-pipelines/gpu-tests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
     - bash: |
         python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
-        pip install fairscale==0.4.0
+        pip install fairscale==0.4.5
```

Comment:
Should we pin its version at the latest one `0.4.6`, or should we simply unpin it to always use the latest version? (I'm curious why we need to pin its version in the first place :eyes:)

Reply:
I think there is compatibility issue. The shard_on_root behavior has changed, I haven't figure out a way to fix the test yet